### PR TITLE
Added some more decomps + fixed some other minor things

### PR DESCRIPTION
**Pull request by Chillee**

### CODE REVIEW

File: torchinductor/lowering.py [hunk 1]
```diff
@@ -929,10 +929,8 @@ def inner_fn(index):
 make_fallback(aten._fused_moving_avg_obs_fq_helper)
 make_fallback(aten.grid_sampler_2d_backward)
 make_fallback(aten.im2col)
-make_fallback(aten.native_group_norm_backward)
 make_fallback(aten.randperm)
 make_fallback(aten.sort)
-make_fallback(aten.sort.stable)
```
Comment (jansel) [inline torchinductor/lowering.py:935]: Was this intentional?

Comment (Chillee) [reply torchinductor/lowering.py:935]: no

File: benchmarks/microbenchmarks/operatorbench.py [hunk 2]
```diff
@@ -179,6 +179,7 @@ def benchmark(suite, op, dtype, max_samples, accuracy_checking):
             except Exception as e:
                 print(f"error {operator}")
                 print(e)
+                raise e
```
Comment (jansel) [inline benchmarks/microbenchmarks/operatorbench.py:182]: if this is intentional remove the `pass` on the next line

**Review by jansel — APPROVED** [review]

File: benchmarks/microbenchmarks/operatorbench.py (hunk 2 above)
Comment (Chillee) [reply benchmarks/microbenchmarks/operatorbench.py:182]: done

File: torchinductor/decomposition.py [hunk 3]
```diff
@@ -20,12 +20,14 @@
...
         aten.addcmul,
         aten.avg_pool2d_backward,
+        aten.binary_cross_entropy_with_logits,
         aten.clamp_max,
         aten.clamp_min,
         aten.col2im_backward,
         aten.cudnn_batch_norm,
         aten.cudnn_batch_norm_backward,
         aten.detach,
+        aten.dot,
```
Comment (ngimel) [inline torchinductor/decomposition.py:30]: but this hasn't landed yet?

Comment (Chillee) [reply torchinductor/decomposition.py:30]: yeah but if it doesn't exist then it won't get it lol.

There's a bunch of decomps in here that don't exist :P

**Pull request merged**